index.html

```diff
@@ -7,7 +7,25 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <div class="container"></div>
+    <div class="container">
+      <div class="title">대기중
+        <span>(0)</span>
+      </div>
+      <div class="content" id="content-incomplete">
+        <!-- <div class="todo">
+          <span>todo</span>
+          <img src="img/bin.png" alt="" class="trashcan">
+        </div> -->
+      </div>
+      <div class="title">완료됨
+        <span>(0)</span>
+      </div>
+      <div class="content"></div>
+      <div class="add">
+        <input type="text" placeholder="새로운 할일 추가" id="new-to-do" />
+        <button class="add_button" onclick="addToDo()">+</button>
+      </div>
```

Comment:
```suggestion
      <form class="add">
        <input type="text" placeholder="새로운 할일 추가" id="new-to-do" />
        <button class="add_button" onclick="addToDo()">+</button>
      </form>
```

저는 개인적으로 `<input>`이랑 `<button>`은 `<form>`안에 넣으시는게 조금 더 낫지 않을까 생각해봅니다..!
`<form>`안에 `<button>`은  기본적으로 form을 submit하는 역할을 하기 때문에 JS 작성시에도 이벤트리스너 함수 코딩하시기에 더 편하실거에요!